docs/en/concepts-and-designs/service-hierarchy.md

```diff
@@ -162,6 +164,22 @@ If you want to customize it according to your own needs, please refer to [Servic
   - VIRTUAL_MQ.service.name: `rabbitmq.skywalking-showcase.svc.cluster.local:5672`
   - RABBITMQ.service.name: `rabbitmq::rabbitmq.skywalking-showcase`
 
+#### KAFKA On K8S_SERVICE
+- Rule name: `short-name`
+- Groovy script: `{ (u, l) -> u.shortName == l.shortName }`
+- Description: KAFKA.service.shortName == K8S_SERVICE.service.shortName
+- Matched Example:
+  - ROCKETMQ.service.name: `kafka::kafka.skywalking-showcase`
+  - K8S_SERVICE.service.name: `skywalking-showcase::kafka.skywalking-showcase`
+
+#### VIRTUAL_MQ On KAFKA
+- Rule name: `lower-short-name-with-fqdn`
+- Groovy script: `{ (u, l) -> u.shortName.substring(0, u.shortName.lastIndexOf(':')) == l.shortName.concat('.svc.cluster.local') }`
+- Description: VIRTUAL_MQ.service.shortName remove port == KAFKA.service.shortName with fqdn suffix
+- Matched Example:
+  - VIRTUAL_MQ.service.name: `kafka.skywalking-showcase.svc.cluster.local:9092`
+  - ROCKETMQ.service.name: `kafka::rocketmq.skywalking-showcase`
```

Comment:
Kafka

Reply:
> Kafka

![KBJ~GJ329(0Z1WJU`E09NVI](https://github.com/apache/skywalking/assets/96932958/319dd262-7007-47d8-b31d-959cf6dfb809)

The config use the lower mapping, Should it be consistent?

Reply:
I mean you put `ROCKETMQ` in here.